src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/convolution_gpu_bfyx_f16_depthwise.cl

```diff
@@ -246,7 +245,11 @@ KERNEL(convolution_depthwise)(
 #else
             res = TO_OUTPUT_TYPE8(dst);
 #endif // HAS_FUSED_OPS
+#if X_BLOCK_SIZE >= 8
             OUTPUT_BLOCK_WRITE8(output, output_offset + x * output_x_pitch, res);
+#else
+            OUTPUT_BLOCK_WRITE(output, output_offset + x * output_x_pitch, res[0]);
```

Comment:
I think if block size is 1, output type should not be OUTPUT_TYPE8 but just OUTPUT_TYPE

Reply:
Also the entire flow is redundant for block_size 1 case. 
Original (only blocksize8 exists)
- if 8 elements available :  block write 
- else : write leftovers 

I think now we can change it like 
#if blocksize ==8
- above code
#else
- write one data only 

To reduce the generated code, might be slightly better for performance too.